vocode/streaming/synthesizer/base_synthesizer.py

```diff
@@ -26,6 +26,14 @@
 FILLER_AUDIO_PATH = os.path.join(os.path.dirname(__file__), "filler_audio")
 TYPING_NOISE_PATH = "%s/typing-noise.wav" % FILLER_AUDIO_PATH
 
+def dtmf_wav_path(digit: str) -> str:
+    # Sounds come from here: https://freesound.org/people/ialexs/packs/10184/
+    # The original audio files didn't work with the wave library, it gave this error:
+    #   wave.Error: unknown format: 65534
+    # The fix was to uncompress them using the sox commandline tool:
+    #   sox 164396__ialexs__dtmf-0.wav -b 16 dtmf-0.wav
```

Comment:
I'm not sure this is the best way to convert the audio file.  I took the suggestion from here:
https://github.com/jiaaro/pydub/issues/129#issuecomment-948220528

It works in the sense that I _can_ hear the file play using the play_dtmf.py playground script.  But it may not be optimal.